src/journal/compress.c

```diff
@@ -18,6 +18,7 @@
   along with systemd; If not, see <http://www.gnu.org/licenses/>.
 ***/
 
+#include <assert.h>
```

Comment:
I think this is not necessary, as `assert()` is provided by macro.h.

Reply:
I needed static_assert()

Reply:
Oh, I see.